Server/htdocs/AppController/commands_RSM/api/v2/items/get.php

```diff
@@ -0,0 +1,174 @@
+<?php
+//***************************************************************************************
+// Description:
+//    Get one or multiple item/s of the specified itemType with the associated values
+// REQUEST BODY (JSON OBJECT):
+//  EXAMPLE 1:
+// {
+//     'IDs': [571],
+//     'itemTypeID': 8
+// }
+//  EXAMPLE 2:
+// {
+//     'IDs': [571, 569],
+//     'propertyIDs': [58,59]
+// }
+//  EXAMPLE 3:
+// {
+//     'propertyIDs': [59],
+//     'filtersRules':
+//      [
+//          {
+//              'propertyID': 58,
+//              'value': 'Sergio',
+//              'operation': '='
+//          }.
+//          {
+//              'propertyID': 59,
+//              'value': 'Santamaria',
+//              'operation': '<>'
+//          }
+//      ]
+// }
+//***************************************************************************************
+
+require_once '../../../utilities/RStools.php';
+require_once '../../../utilities/RSMverifyBody.php';
+setAuthorizationTokenOnGlobals();
+checkCorrectRequestMethod('GET');
+
+require_once '../../../utilities/RSdatabase.php';
+require_once '../../../utilities/RSMitemsManagement.php';
+require_once '../../api_headers.php';
+
+// Definitions
+$requestBody = getRequestBody();
+verifyBodyContent($requestBody);
+
+$clientID = getClientID();
+$RStoken =  getRStoken();
+$RSuserID =  getRSuserID();
+
+// Params
+$propertyIDs = $requestBody->propertyIDs;
+$filterRules = $requestBody->filterRules;
+$extFilterRules = $requestBody->extFilterRules;
+$IDs = $requestBody->IDs;
+$itemTypeID = $requestBody->itemTypeID;
+
+// includeCategories filter
+$includeCategories = false;
+if (isset($requestBody->includeCategories) && $requestBody->includeCategories) {
+  $includeCategories = true;
+}
+
+// translateIDs
+$translateIDs = false;
+if (isset($requestBody->translateIDs) && $requestBody->translateIDs) {
+  $translateIDs = true;
+}
+
+// itemTypeID
+if ($itemTypeID == '') {
+  $itemTypeID = getItemTypeIDFromProperties($propertyIDs, $clientID);
+}
+
+if ($itemTypeID <= 0) {
+  $RSallowDebug ? returnJsonMessage(400, 'Invalid itemTypeID: ' . $itemTypeID) : returnJsonMessage(400, '');
+}
+
+//propertyIDs
+if ($propertyIDs == '') {
+  $propertyIDs = getClientItemTypePropertiesId($itemTypeID, $clientID);
+}
+
+// IDs
+if ($IDs != '') {
+  $IDs = implode(',', $IDs);
+}
```

Comment:
In line 86, instead of comparing IDs with an empty text, which would throw an error or warning if IDs is an array,, I would use the is_array  function to check if IDs is an array before doing the implode. If IDs is an empty array, the implode would also return an empty string, which is also OK.